src/Simplex/Messaging/Agent/Client.hs

```diff
@@ -1404,6 +1405,9 @@ subscribeQueues c qs = do
       let (userId, srv, _) = transportSession' smp
       atomically $ incSMPServerStat' c userId srv connSubAttempts (length qs')
       rs <- sendBatch subscribeSMPQueues smp qs'
+      let (successes, errs) = partition (isRight . snd) (L.toList rs)
+      atomically $ incSMPServerStat' c userId srv connSubscribed (length successes)
+      atomically $ incSMPServerStat' c userId srv connSubErrs (length errs)
```

Comment:
there are temporary errors here, that are accounted for in attempts - this should only account for permanent errors